src/test/unit/requests/statusDescriptionRequests.test.ts

```diff
@@ -4,60 +4,58 @@ import {StatusDescriptionRequests} from '../../../main/resources/requests/status
 import fs from 'fs';
 import path from 'path';
 
-const statusDescriptionRequests = new StatusDescriptionRequests();
+const searchDescriptionRequests = new StatusDescriptionRequests();
```

Comment:
it was right before

Reply:
the name of this file needs changing too

Reply:
in fact statusDescription is wrong too, this is supposed to be the event glossary right? so where did statusDescription come from?

Reply:
I don't remember where it came from. Do you want me to change all the relavent files to EventGlossary or CaseEventGlossary or CourtEventGlossary or CaseStatusDescription or something else? Reference link http://xhibit.justice.gov.uk/statusdescription.html

Reply:
i think either CaseEventGlossary or EventGlossary will be fine

Reply:
everything change to CaseEventGlossary.